.env

```diff
@@ -117,7 +117,7 @@ export SVIX_CALLBACK_URL=http://bats-tests:8080/webhook/
 export KRATOS_PUBLIC_API="http://localhost:4433"
 export KRATOS_ADMIN_API="http://localhost:4434"
 
-# export IBEX_URL="https://api-sandbox.poweredbyibex.io"
+export IBEX_URL="https://api-sandbox.poweredbyibex.io"
```

Comment:
did you mean to leave this uncommented?

Reply:
Yeah, this should be the default Ibex URL we use when running locally.